engine/android-gradle/app/.externalNativeBuild/ndkBuild/debug/armeabi-v7a/android_gradle_build_mini.json

```diff
@@ -0,0 +1,23 @@
+{
+  "buildFiles": [
+    "/home/haxpor/Data/Projects/abzico/openbor/engine/android-gradle/app/jni/Android.mk",
```

Comment:
This and the file under it have references to your computer's home directory. Could they work with relative paths, or are they perhaps auto-generated files that doesn't need to be committed?

Reply:
Ohh, ok, thank you @Plombo !, that is definitely me overlooking files that should be ignored. I'll do this in a moment.

Reply:
Those files are generated by gradle as part of build process, I removed and untracked them from git tree now. Tested re-building works fine.

Reply:
Any directories starting with a "." should be deleted before applying to source code.

I finally got this to work on windows.  The issue was you referenced some files from the android folder when I copied the real files over it compiled just fine.  This is not a build that should run along side the old Ant method it should replace it.

There is a issue with the new debug apk it will only install on a device once if you uninstall it then your stuck unless you compile a new one.  This will be a problem for releases but we can fix that later.

@DCurrent this pull is not windows compatible so I suggest rejecting it.

@haxpor again great work with this and if you don't mind can I submit a modified version of this commit ?

Reply:
@msmalik681 sure no problem. I believe you have more portable solution to fix it.But curious to know what's that solution?

Could you elaborate more on what's the problem of installing on device?

Reply:
There was a issue with premissions on android 6.0 upwards.  I have fixed the issue but need to clean up the process before I make a pull request based on this method.

The issue was as we are targeting a higher api the premission request is no longer shown on install we need to make a pop up giving the user the option to accept the required permissions.